### PR TITLE
Add php8 compatibility

### PR DESCRIPTION
**Pull request by patrickharriscatalyst**
I ran phpcs phpcompatability on this module and found no errors or warnings. Can we tentatively add support for PHP8?

### CODE REVIEW

**Comment by RoSk0** [comment]
Thanks!

Can you please also add PHP 8 to the test matrix https://github.com/drupalauth/simplesamlphp-module-drupalauth/blob/master/.travis.yml#L6

**Comment by RoSk0** [comment]
Sadly I discovered that Travis CI no longer builds this project. Unfortunately, until I figure out the direction for CI I'm not in a position to accept this change.

**Comment by RoSk0** [comment]
Now that we have CI back, can you please add 8.1 to Composer as well?

**Comment by patrickharriscatalyst** [comment]
Isn't 8.1 covered by `^8.0` in composer.json?

**Comment by RoSk0** [comment]
Yes, you are right, but explicit is always better than implicit. 

Please add PHP 8.1 to composer.json.

**Comment by patrickharriscatalyst** [comment]
Done!

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ php:
    - '7.2'
    - '7.3'
    - '7.4'
+   - '8.0'
+   - '8.1'
```
Comment (RoSk0) [inline .travis.yml:8]:
```suggestion
   - '8.1.0'
```

It looks like we need full version atm for 8.1.

See https://travis-ci.community/t/php-8-1-support/12439/23 for details

File: .travis.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ php:
    - '7.2'
    - '7.3'
    - '7.4'
+   - '8.0'
+   - '8.1.0'
```
Comment (RoSk0) [inline .travis.yml:8]:
```suggestion
```

Looks like Travis is not completely ready for 8.1 so we are not going to rush either.

File: composer.json [hunk 3]
```diff
@@ -21,7 +21,7 @@
     }
   ],
   "require": {
-    "php": "^5.6.0|^7.0",
+    "php": "^5.6.0|^7.0|^8.0|^8.1",
```
Comment (RoSk0) [inline composer.json:24]:
```suggestion
    "php": "^5.6.0|^7.0|^8.0",
```

**Review by RoSk0 — CHANGES_REQUESTED** [review]

**Comment by RoSk0** [comment]
Thanks Patrick!

**Pull request merged**